Wiki/Modules/FvwmBacker/index.md

```diff
@@ -3,14 +3,94 @@ title: FvwmBacker
 type: module
 weight: 430
 description: |
-  This module will change the backgound/wallpaper when changing desktops.
+  This module will change the background/wallpaper when changing desktops.
   Any command can be executed when changing desktops so this module can
   be used for more than just switching the background.
 ---
 # FvwmBacker
 
-The FvwmBacker module provides functionality to change the  background when
-changing  desktops.  Any command can be executed to change the backgrounds.
-Actually, any arbitrary command can be sent to fvwm to execute, so  you
+* TOC
+{:toc}
+
+The FvwmBacker module provides functionality to change the background when
+changing  desktops. Any command can be executed to change the backgrounds.
+Actually, any arbitrary command can be sent to fvwm to execute, so you
 could also do things such as changing window border colors, etc.
 
+You can use the FvwmPager to move from screen to screen through each virtual
+desktop set. Now that you can move easily between the desktops you can further
+personalize them by giving them individual background colors.
+
+{% fvwm2rc %}
+*FvwmBacker: Command (Desk 2) -solid white
+{% endfvwm2rc %}
+
+The changes will not be implemented until you instruct Fvwm to update the screens
+during the initialization:
+
+{% fvwm2rc %}
+AddToFunc InitFunction I Module FvwmBacker
+{% endfvwm2rc %}
+
+Instead of colors, you can define background images:
+
+{% fvwm2rc %}
+*FvwmBacker: Command (Desk 0) Exec exec fvwm-root $HOME/background-mono.png
+*FvwmBacker: Command (Desk 1) Exec exec fvwm-root $HOME/wallpaper-nature.jpg
+{% endfvwm2rc %}
```

Comment:
Maybe add another example to state that, not only can FvwmBacker give each desktop a unique look, it can also be used to change the background for each Page as well, and then provide some examples.